.gitignore

```diff
@@ -16,7 +16,9 @@ arelle/plugin/validate/eforms.py
 arelle/plugin/validate/ESEF-DQC.py
 arelle/plugin/xule
 Arelle.egg-info
+arelleGUI.py
 .eggs
+.python-version
```

Comment:
Not strictly necessary, but ran into this while testing the Linux docker builds locally (global gitignore isn't present in docker context).